packages/expo-background-task/src/ExpoBackgroundTaskModule.ts

```diff
@@ -0,0 +1,13 @@
+import { requireNativeModule } from 'expo-modules-core';
+import type { NativeModule } from 'expo-modules-core';
```

Comment:
```suggestion
import { requireNativeModule, type NativeModule } from 'expo';
```

these are re-exported from the expo package